sklearn/ensemble/forest.py

```diff
@@ -758,6 +763,10 @@ class RandomForestClassifier(ForestClassifier):
     n_estimators : integer, optional (default=10)
         The number of trees in the forest.
 
+        .. deprecated:: 0.20
```

Comment:
should be "versionchanged" not "deprecated"

Reply:
versionchanged as one long word, no spaces?

Reply:
yes. ``git grep versionchanged``?

Reply:
working on that.